rest-api/src/main/java/gov/cms/qpp/conversion/api/model/CpcValidationInfo.java

```diff
@@ -0,0 +1,44 @@
+package gov.cms.qpp.conversion.api.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class CpcValidationInfo {
+	private String npi;
+	private String tin;
+	private String apm;
+
+	CpcValidationInfo() {
+		// Empty Constructor for Jackson
+	}
+
+	CpcValidationInfo(String npi, String tin, String apm) {
+		this.npi = npi;
+		this.tin = tin;
+		this.apm = apm;
+	}
+
+	public String getNpi() {
+		return npi;
+	}
+
+	public void setNpi(final String npi) {
+		this.npi = npi;
+	}
+
+	public String getTin() {
+		return tin;
+	}
+
+	public void setTin(final String tin) {
+		this.tin = tin;
+	}
+
+	@JsonProperty("apm_entity_id")
+	public String getApm() {
+		return apm;
+	}
+
+	public void setApm(final String apm) {
+		this.apm = apm;
+	}
+}
```

Comment:
While this is a POJO, unit test coverage has been added anyhow.

Reply:
Yes. I believe these would have been covered in other unit tests that use this object. Thanks for the help though.